bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeSystem.scala

```diff
@@ -1,21 +1,27 @@
 package edu.gemini.spModel.core
 
+import scala.collection.JavaConversions._
 import scalaz.Equal
 
 sealed abstract class MagnitudeSystem(val name: String) extends Product with Serializable
 
 object MagnitudeSystem {
 
-  case object VEGA extends MagnitudeSystem("Vega")
-  case object AB   extends MagnitudeSystem("AB")
-  case object JY   extends MagnitudeSystem("Jy")
+  case object Vega   extends MagnitudeSystem("Vega")
```

Comment:
Renamed in order to make sure that `toString()` yields the same value as before.